README.md

```diff
@@ -1 +1,73 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Capstone
+
+> in this project:
+- Set up the linters for html, css, and JavaScript.
+- Create a home and about pages
+- Add cards dynamically through JavaScript
+
+
+## Built With 
+
+- Html
+- Css
+- JavaScript
+- VS Studio
+- Git Bash
+
+## Live Demo (if available)
+
+[Click](https://ibr5500.github.io/Capstone/) for Live Demo Link
+
```

Comment:
- [ ] Kindly record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

- A description of the project.
- A demo of the project features (different pages and different screen sizes).
- You should also highlight some interesting piece of code or something you built that you are very proud of.